### ISSUE TITLE
FRP: Add keepVELO to strategy page fee UI for Velodrome vaults

### ISSUE THREAD

**Issue by dudesahn** [yearn/yearn.fi#310]
### What is your proposal 🚀
**A clear and concise description of what you are asking for.**

- To the right here, add a field to visualize our keepVELO percentage. This can be pulled from exporter in the `fees`, if it is a velodrome vault then it will report a `keep_velo` percentage (between 0 and 1) once this PR is merged in their repo: https://github.com/yearn/yearn-exporter/pull/568
- Add a hover tooltip that explains what keepVELO is. Something like "Percentage of VELO locked in each harvest. This is used to boost vault pools, and is offset via yvOP staking rewards."

<img width="640" alt="Screen Shot 2023-08-01 at 8 29 09 AM" src="https://github.com/yearn/yearn.fi/assets/23222916/543ba050-e8ed-4887-96df-5f52bc2b2ade">

### What does this solve ? 🛠
**A clear and concise description about why this is a nice to have.**

- Although we already include keepVELO in our overall APR calc, it's nice that users can know exactly how much velo is being kept by a vault. And the tooltip lets them know what it is and why we do it, and tells them they are getting more than enough to cover it in yvOP.

### More 🔎
*N/A*
